README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://travis-ci.org/dbtsai/python-mimeparse.svg?branch=master
+   :target: https://travis-ci.org/dbtsai/python-mimeparse
+
+This module provides basic functions for handling mime-types. It can
+handle matching mime-types against a list of media-ranges. See section
+5.3.2 of the HTTP 1.1 Semantics and Content specification [RFC 7231] for
+a complete explanation: https://tools.ietf.org/html/rfc7231#section-5.3.2
+
+Functions
+=========
+
+``parse_mime_type()``
+----------------------
+
+Parses a mime-type into its component parts.
+
+``parse_media_range()``
+-----------------------
+
+Media-ranges are mime-types with wild-cards and a "q" quality parameter.
+
+``quality()``
+-------------
+
+Determines the quality ("q") of a mime-type when compared against a list of
+media-ranges.
+
+``quality_parsed()``
+--------------------
+
+Just like ``quality()`` except the second parameter must be pre-parsed.
+
+``best_match()``
+----------------
+
+Choose the mime-type with the highest quality ("q") from a list of candidates.
+
+Testing
+=======
+
+The format of the JSON test data file is as follows: A top-level JSON object
+which has a key for each of the functions to be tested. The value corresponding
+to that key is a list of tests. Each test contains: the argument or arguments
+to the function being tested, the expected results and an optional description.
+
+Python
```

Comment:
What do you think about dropping the "Python" section and instead adding this above the "Functions" section:

> Installation
> =======
> 
> To install this package, simply use `pip install python-mimeparse`. It supports Python 2.7 - 3.5 and PyPy.